lib/travis/api/app.rb

```diff
@@ -198,11 +198,11 @@ def self.setup_travis
       end
 
       def self.setup_database_connections
-        Travis.config.database.variables                  ||= {}
-        Travis.config.database.variables.application_name ||= ["api", Travis.env, ENV['DYNO']].compact.join(?-)
+        Travis.config.database.variables                    ||= {}
+        Travis.config.database.variables[:application_name] ||= ["api", Travis.env, ENV['DYNO']].compact.join(?-)
         Travis::Database.connect
 
-        if Travis.config.logs_database
+        if Travis.config.logs_database.to_h
```

Comment:
I think Hashr allows us to do `Travis.config.logs_database?` to check if the key is set.